src/client/dfuse/dfuse_main.c

```diff
@@ -281,39 +281,39 @@ show_help(char *name)
 int
 main(int argc, char **argv)
 {
-	struct dfuse_projection_info	*fs_handle = NULL;
-	struct dfuse_info	*dfuse_info = NULL;
-	struct dfuse_pool	*dfp = NULL;
-	struct dfuse_cont	*dfs = NULL;
-	struct duns_attr_t	path_attr = {};
-	struct duns_attr_t	duns_attr = {};
-	uuid_t			cont_uuid = {};
-	uuid_t			pool_uuid = {};
-	char			*pool_name = NULL;
-	char			*cont_name = NULL;
-	char			c;
-	int			rc;
-	int			rc2;
-	char			*path = NULL;
-	bool			have_thread_count = false;
-
-	struct option long_options[] = {
-		{"mountpoint",		required_argument, 0, 'm'},
-		{"path",		required_argument, 0, 'P'},
-		{"pool",		required_argument, 0, 'p'},
-		{"container",		required_argument, 0, 'c'},
-		{"sys-name",		required_argument, 0, 'G'},
-		{"singlethread",	no_argument,	   0, 'S'},
-		{"thread-count",	required_argument, 0, 't'},
-		{"foreground",		no_argument,	   0, 'f'},
-		{"enable-caching",	no_argument,	   0, 'E'},
-		{"enable-wb-cache",	no_argument,	   0, 'F'},
-		{"disable-caching",	no_argument,	   0, 'A'},
-		{"disable-wb-cache",	no_argument,	   0, 'B'},
-		{"version",		no_argument,	   0, 'v'},
-		{"help",		no_argument,	   0, 'h'},
-		{0, 0, 0, 0}
-	};
+	struct dfuse_projection_info *fs_handle  = NULL;
+	struct dfuse_info            *dfuse_info = NULL;
+	struct dfuse_pool            *dfp        = NULL;
+	struct dfuse_cont            *dfs        = NULL;
+	struct duns_attr_t            path_attr  = {};
+	struct duns_attr_t            duns_attr  = {};
+	uuid_t                        cont_uuid  = {};
+	uuid_t                        pool_uuid  = {};
+	char                         *pool_name  = NULL;
+	char                         *cont_name  = NULL;
+	char                          c;
+	int                           rc;
+	int                           rc2;
+	char                         *path              = NULL;
+	bool                          have_thread_count = false;
+	int                           pos_index         = 0;
+
+	struct option                 long_options[] = {{"mountpoint", required_argument, 0, 'm'},
+							{"path", required_argument, 0, 'P'},
+							{"pool", required_argument, 0, 'p'},
+							{"container", required_argument, 0, 'c'},
+							{"sys-name", required_argument, 0, 'G'},
+							{"singlethread", no_argument, 0, 'S'},
+							{"thread-count", required_argument, 0, 't'},
+							{"foreground", no_argument, 0, 'f'},
+							{"enable-caching", no_argument, 0, 'E'},
+							{"enable-wb-cache", no_argument, 0, 'F'},
+							{"disable-caching", no_argument, 0, 'A'},
+							{"disable-wb-cache", no_argument, 0, 'B'},
+							{"options", required_argument, 0, 'o'},
```

Comment:
I see what you mean now, I thought it would be silently dropped but reading the code it should be rejected which means the mount would fail.  Investigating now.